### PR TITLE
feat: add calculator to pricing page

### PR DESCRIPTION
**Pull request by bryanoltman**
Adds a pricing calculator to the pricing page:

https://github.com/user-attachments/assets/47e5f087-eb3c-4926-b47d-63f59d55f1e6

### CODE REVIEW

File: src/components/PricingCalculator.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            We recommend the{' '}
+            <span className="font-bold">{recommendedPlan}</span> plan
+            {cost !== null && (
+              <span className="ml-2 text-sm">
+                ({formatMoney(cost)} per month)
+              </span>
+            )}
+          </div>
+          <div className="text-m mt-2 max-w-2xl">
+            You need a plan with{' '}
```
Comment (eseidel) [inline src/components/PricingCalculator.tsx:164]:
```suggestion
            You will want a plan with{' '}
```

**Comment by eseidel** [comment]
Feedback:
1. You might want sliders to only stop at the few chosen stops (rather than continous).
2. In general language should show these as "at most" costs rather than "it will cost this".
3. If they're patching more than 4 times a month, we might want to show a line saying how costs go down as patch frequency goes up, as we only charge you for isntalled patches, and we won't install patches while users are not using the app.

**Review by eseidel — APPROVED** [review]

**Comment by bryanoltman** [comment]
As discussed on VC, landing and applying the feedback above in future PRs

**Comment by eseidel** [comment]
Feedback after trying:
1. Remove "Use our pricing calculator to find out."
2. "How many monthly active users do you have?" -> "Approximate Monthly Active Users"
3. "Our Pro plan" -> "Pro" or "The Pro Plan"?
4. "We recommend the" -> "Use the" or "Try the"? to be shorter?
5. "Sign up here." should be its own separate line?
6. Should "*Prices are quoted in USD and sold as "patch installs per month", reflecting successful installs of a given patch. For example, 1 patch pushed to 10 devices is 10 installs. 2 patches pushed to 5 devices is also 10 installs." be above the calculator?

**Pull request merged**